docusaurus/docs/iOS/swiftui/components/helper-views.md

```diff
@@ -94,7 +94,7 @@ In the channel list item creation method, you are provided with several paramete
 - `avatar`: the avatar of the channel.
 - `onlineIndicatorShown`: whether the online indicator (about last active members) is shown on the avatar.
 - `disabled`: whether the user interactions with the channel are disabled. You should use this value while the view is being swiped, in order to avoid clicking the channel list item instead.
-- `selectedChannel`: binding of the currently selected channel.
+- `selectedChannel`: binding of the currently selected channel selection info (channel and optional message).
```

Comment:
Maybe this is better like this? **selected channel selection info** sounds a bit weird and repetitive  🤔 
```suggestion
- `selectedChannel`: binding of the current channel selection info (channel and optional message).
```